fusillade-api.yml

```diff
@@ -1282,6 +1282,99 @@ paths:
           $ref: '#/components/responses/404'
         default:
           description: Unexpected error.
+  /v1/resource/{resource_type_name}/id:
+    get:
+      summary: list all ids of a resource type.
+      description: List all resource id of the specified resource type.
+      operationId: fusillade.api.resource.resource_type_name.id.get
+      security:
+        - BearerAuth: []
+      parameters:
+        - $ref: '#/components/parameters/resource_type_name'
+        - $ref: '#/components/parameters/next_token'
+        - $ref: '#/components/parameters/per_page'
+      responses:
+        200:
+          $ref: '#/components/responses/200_paged'
+        206:
+          $ref: '#/components/responses/206'
+        401:
+          $ref: '#/components/responses/401'
+        403:
+          $ref: '#/components/responses/403'
+        404:
+          $ref: '#/components/responses/404'
+        default:
+          description: Unexpected error.
+      tags:
+        - resource
+  /v1/resource/{resource_type_name}/id/{resource_id}:
+    get:
+      summary: check if a resource exists.
```

Comment:
This summary needs a bit of clarification. For example, what if I specify a valid resource ID that I know exists, but the resource type I specify does not match the type of that resource ID? I would expect Fusillade to raise an error in that case, but then the endpoint is not just checking if a resource exists.

Maybe something like `Check if a resource of type resource_type_name with id resource_id exists`

Reply:
the naming in the openapi spec is getting out of hand and needs to be cleaned up. Look for a refactor in the near future.